src/checks/unordered_key.rs

```diff
@@ -27,8 +27,8 @@ impl Default for UnorderedKeyChecker<'_> {
 
 impl Check for UnorderedKeyChecker<'_> {
     fn run(&mut self, line: &LineEntry) -> Option<Warning> {
-        // Support of grouping variables through blank lines
-        if line.is_empty() {
+        // Support of grouping variables through blank lines and control comments
+        if line.is_empty() || line.get_comment().is_some() {
```

Comment:
```suggestion
        if line.is_empty() || line.is_comment() {
```
This check should also support usual comments.

Reply:
Done